exp/views/dashboard.py

```diff
@@ -14,6 +14,10 @@ class ExperimenterDashboardView(ExperimenterLoginRequiredMixin, generic.Template
     template_name = 'exp/dashboard.html'
 
     def dispatch(self, request, *args, **kwargs):
-        if self.request.user.groups.exists():
-            return redirect(reverse_lazy('exp:study-list'))
-        return super().dispatch(request, *args, **kwargs)
+        if self.request.path.endswith('/'):
```

Comment:
@cwisecarver so this PR also added similar logic to the proxy view.   The reason I added things to the dashboard view is that experimenter side, too many things were getting redirected to manage studies from the dashboard view if there was a problem.  I know it's not so proper, but the trailing slash thing seems to be working well across all views with this and the other fix.